packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -429,6 +429,36 @@ describe('Connection screen', function () {
     expect(result).to.have.property('ok', 1);
   });
 
+  it('can connect to an Atlas with tlsUseSystemCA', async function () {
+    if (!hasAtlasEnvironmentVariables()) {
+      //return this.skip();
```

Comment:
Is this part commented intentionally? If yes maybe `if` can be deleted.

Reply:
No, it’s not commented intentionally, thanks for pointing this out!